src/structures.rs

```diff
@@ -241,7 +236,15 @@ impl fmt::Display for SvParameter {
                 writeln!(f, "    Signedness: {:?}", x)?;
             }
         }
-        writeln!(f, "    SignednessStatus: {:?}", self.signedness_status)?;
+        writeln!(f, "    SignednessStatus: {:?}", self.signedness_overridable)?;
```

Comment:
SignednessStatus -> SignednessOverridable

Reply:
Thank you for pointing it out. I have already fixed it the latest bugfixes committed in the "expression-resolver" branch and I will update it here as well.

Reply:
Fixed.